src/wemmick/utils.py

```diff
@@ -14,3 +15,36 @@ def file_relative_path(dunderfile, relative_path: str):
     H/T https://github.com/dagster-io/dagster/blob/8a250e9619a49e8bff8e9aa7435df89c2d2ea039/python_modules/dagster/dagster/utils/__init__.py#L34
     """
     return os.path.join(os.path.dirname(dunderfile), relative_path)
+
+
+def get_scheme(path: str):
+    return urlparse(path).scheme
+
+
+def check_file_exists(path):
+    if not os.path.isfile(path):
+        raise ValueError(
+            f"File {path} was not found. Please check the path and try again."
+        )
+
+
+def get_file(path: str):
+    scheme = get_scheme(path)
+    if scheme == "s3":
+        # TODO: add AWS S3 support
+        pass
+    elif scheme == "gs":
+        # TODO: add GCP Bucket support
+        pass
+    elif scheme == "file":
+        file_path = urlparse(path).path
```

Comment:
I think it would be better if this logic was its own function outside of the elif

Reply:
Will refactor this

Reply:
Cleaned this up.